sentry-android-core/src/main/java/io/sentry/android/core/ActivityFramesTracker.java

```diff
@@ -104,9 +102,9 @@ public synchronized void setMetrics(
       return;
     }
 
-    final MeasurementValue tfValues = new MeasurementValue(totalFrames, NONE_UNIT);
-    final MeasurementValue sfValues = new MeasurementValue(slowFrames, NONE_UNIT);
-    final MeasurementValue ffValues = new MeasurementValue(frozenFrames, NONE_UNIT);
+    final MeasurementValue tfValues = new MeasurementValue(totalFrames, null);
+    final MeasurementValue sfValues = new MeasurementValue(slowFrames, null);
+    final MeasurementValue ffValues = new MeasurementValue(frozenFrames, null);
```

Comment:
We default to `none` on Flutter, and RN.
Any reason to change it?

Reply:
See last message 🤣 
![image](https://user-images.githubusercontent.com/4999776/193563224-bc84a657-2240-426a-882e-700712e3b3bd.png)

Reply:
or was it only meant for the custom custom measurements?

Reply:
forget it, I just re-read the thread and it makes sense to have `none` here. thanks for pointing out!